atomicapp/providers/openshift.py

```diff
@@ -313,6 +365,9 @@ def _parse_kubeconf(self, filename):
             if co["name"] == current_context:
                 context = co
 
+        if not context:
+            raise ProviderFailedException()
```

Comment:
got ya. you're right

It would still be nice to print out the Invalid filename like was done on line 327 before?